### ISSUE TITLE
Remove all `from X import *`

### ISSUE THREAD

**Issue by LordGolias** [LordGolias/Brytenwalda#3]
It is a bad practice to use this idiom in Python because it populates the namespace. I'm suspecting that the reason why the compilation takes so much time is because it is running lots of code for every `from X import *` that appears in every module.

**Comment by gdwitt** [comment]
I'm not sure I understand "Populates the namespaece". This is a 4 year old build system build on a 7 year old system. You could look at some more recent mods and find a quicker method. Most developers are not open source like this one. For me, WRECK makes things simple and quick enough for me. I would like to update some scripts tonite to fix a bug. Can I just sync from my desktop git?

**Comment by LordGolias** [comment]
So, I've finished this and issue #2 by writing a compiler that super-seeds WRECK and the `process_*`.

The source code now consists of 2 Python packages: `source`, that contains `module_*` and `header_*`, and `compiler` that contains the code to compile all `.txt`. The code in commit 03de6a4011f reproduces all the `.txt` in version 23 of Britenwalda.

The compiler allows to compile the source without `ID_*.py`, that were unneeded. Every time `ID_*` was required by a `module_*`, (e.g. to put the id of `fac_neutral`), the code was replaced by `"fac_neutral"` (a string). The compiler then takes care of replacing the string by the correct id on the final code, which is the main goal of a compiler anyway.

This code also got us rid of WRECK, that was really difficult to read, and of all those `process_*` that saved files just by getting imported.

I've updated the README with the instructions to compile.

Meanwhile, this new compiler allowed to find several errors on the code that probably deserve our attention. For example, there are two scripts or triggers that are missing `try_end`s.

```
CRITICAL:root:Statement "0" misses 1 "try_end" statement_blocks
CRITICAL:root:Statement "0" has 1 "try_end" too many
```
